### PR TITLE
refactor: prev->newer next->older

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Rename the buttons and related functions of ProposalNavigation component to improve code readability.

# Changes

- prev -> newer
- next -> older

# Todos

- [ ] Add entry to changelog (if necessary).

### CODE REVIEW

File: frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte [hunk 1]
```diff
@@ -4,26 +4,27 @@
   import { assertNonNullish, isNullish } from "@dfinity/utils";
 
   export let currentProposalId: bigint;
+  // Filtered proposal ids sorted in descending order
   export let proposalIds: bigint[] = [];
```
Comment (dskloetd) [inline frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte:8]: What do you think about throwing an exception if this is not in descending order?
Or sorting to make sure it's in descending order?

File: frontend/src/tests/lib/components/proposal-detail/ProposalNavigation.spec.ts [hunk 2]
```diff
@@ -65,8 +65,8 @@ describe("ProposalNavigation", () => {
         selectProposal: jest.fn(),
       });
 
-      expect(await po.isNextButtonHidden()).toBe(false);
-      expect(await po.isPreviousButtonHidden()).toBe(false);
+      expect(await po.isOlderButtonHidden()).toBe(false);
+      expect(await po.isNewerButtonHidden()).toBe(false);
     });
 
     it("should disable previous button when it's selected", async () => {
```
Comment (dskloetd) [inline frontend/src/tests/lib/components/proposal-detail/ProposalNavigation.spec.ts:72]: Fix this description as well?

File: frontend/src/tests/lib/components/proposal-detail/ProposalNavigation.spec.ts [hunk 3]
```diff
@@ -76,8 +76,8 @@ describe("ProposalNavigation", () => {
         selectProposal: jest.fn(),
       });
 
-      expect(await po.isNextButtonHidden()).toBe(false);
-      expect(await po.isPreviousButtonHidden()).toBe(true);
+      expect(await po.isOlderButtonHidden()).toBe(false);
+      expect(await po.isNewerButtonHidden()).toBe(true);
     });
 
     it("should disable next when it's selected", async () => {
```
Comment (dskloetd) [inline frontend/src/tests/lib/components/proposal-detail/ProposalNavigation.spec.ts:83]: Fix this description as well?

File: frontend/src/tests/lib/components/proposal-detail/ProposalNavigation.spec.ts [hunk 4]
```diff
@@ -114,7 +114,7 @@ describe("ProposalNavigation", () => {
       selectProposal: selectProposalSpy,
```
Comment (dskloetd) [inline frontend/src/tests/lib/components/proposal-detail/ProposalNavigation.spec.ts:114]: Fix the description above on line 109?

File: frontend/src/tests/lib/components/proposal-detail/ProposalNavigation.spec.ts [hunk 5]
```diff
@@ -87,8 +87,8 @@ describe("ProposalNavigation", () => {
         selectProposal: jest.fn(),
       });
 
-      expect(await po.isNextButtonHidden()).toBe(true);
-      expect(await po.isPreviousButtonHidden()).toBe(false);
+      expect(await po.isOlderButtonHidden()).toBe(true);
+      expect(await po.isNewerButtonHidden()).toBe(false);
     });
   });
```
Comment (dskloetd) [inline frontend/src/tests/lib/components/proposal-detail/ProposalNavigation.spec.ts:94]: Fix the description below on line 95 as well?

**Review by dskloetd — APPROVED** [review]
Thanks!

File: frontend/src/tests/lib/components/proposal-detail/ProposalNavigation.spec.ts (hunk 2 above)
Comment (mstrasinskis) [reply frontend/src/tests/lib/components/proposal-detail/ProposalNavigation.spec.ts]: Good point, done.

File: frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte:7]: I think this is not a bad idea. Sorting was added.

**Pull request merged**